### PR TITLE
Fix for incorrect attaching of raw data files to email schedules chart report

### PR DESCRIPTION
**Pull request by mikalai-dev**
### CATEGORY

Choose one

- [x] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
Schedule Email Reports fails in a case of attached data files. Present fix resolve this regression from the removal of the dependency `request`

### ADDITIONAL INFORMATION

- [x] Has associated issue: [#8286](https://github.com/apache/incubator-superset/issues/8286)
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### REVIEWERS
@villebro

### CODE REVIEW

File: superset/tasks/schedules.py [hunk 1]
```diff
@@ -281,7 +282,11 @@ def _get_slice_data(schedule):
             )
 
     elif schedule.delivery_type == EmailDeliveryType.attachment:
-        data = {__("%(name)s.csv", name=slc.slice_name): response.content}
+        csv_content = ""
+        for r in content:
+          csv_content += str(r, 'utf-8')
```
Comment (villebro) [inline superset/tasks/schedules.py]: Indentation here needs to be four spaces, not two.

**Review by villebro — CHANGES_REQUESTED** [review]
First pass comments: please fix indentation error and run `black` for uniform code formatting.

**Comment by villebro** [comment]
You still need to run black on the modified files. Furthermore, the tests are failing because the CSV request mock in the test case conforms to the `requests` format, not `urllib` format. Check the following lines: https://github.com/apache/incubator-superset/blob/master/tests/schedules_test.py#L370_L374 and https://github.com/apache/incubator-superset/blob/master/tests/schedules_test.py#L396-L400 . Let me know if you need help modifying them.

File: README.md [hunk 2]
```diff
@@ -214,3 +214,4 @@ the world know they are using Superset. Join our growing community!
  1. [Zalando](https://www.zalando.com)
  1. [Zalora](https://www.zalora.com)
  1. [TME QQMUSIC/WESING](https://www.tencentmusic.com/)
+ 1. [Facemetrics](https://facemetrics.io/)
```
Comment (etr2460) [inline README.md]: nit: abc order please!

**Comment by villebro** [comment]
@ngritsuk can I help you finalize this PR somehow? This is a regression that I would be very happy to see getting merged soon.

**Comment by ngritsuk** [comment]
@villebro Probably I'll have time to deal with tests at the end of this week. I'll be happy if you help me with them.

**Pull request closed without merging**